facebook/__init__.py

```diff
@@ -235,7 +243,7 @@ def request(
             # or it does not need `access_token`.
             if post_args and "access_token" not in post_args:
                 post_args["access_token"] = self.access_token
-            elif "access_token" not in args:
+            elif args and "access_token" not in args:
```

Comment:
Per the [Contributing Guidelines](https://github.com/mobolic/facebook-sdk/blob/master/CONTRIBUTING.rst), a pull request should only contain commits that aren't relevant to its core feature. If it doesn't have to do with user permissions, it should be in a different pull request.

The purpose of that code is to ensure that the arguments passed to Facebook include an access token. With your code, it is necessary to set `args`, otherwise this will not happen.

This should have been fixed in #341, so if you rebase your branch off of the current master, the error will go away.